server/apps/publish/subscribers.py

```diff
@@ -18,6 +20,9 @@
 
 logger = logging.getLogger(__name__)
 
+subscriber_types = ['broadcast', 'digital', 'wire']
+SUBSCRIBER_TYPES = namedtuple('SUBSCRIBER_TYPES', ['BROADCAST', 'DIGITAL', 'WIRE'])(*subscriber_types)
```

Comment:
Wire is correct and for Digital Subscribers, the Takes Package is transmitted not the individual takes. Need to check with business if broadcast subscribers can receive the takes package. Example given by business for a broadcast subscriber is a TV Station or Radio Station.